index.js

```diff
@@ -16,13 +20,21 @@ module.exports = {
 
     if (!process.env.EMBER_CLI_FASTBOOT) {
       if (app.env === "production") {
-        app.import(app.bowerDirectory + '/hammer.js/hammer.min.js');
+        app.import('vendor/hammer.min.js');
       } else {
-        app.import(app.bowerDirectory + '/hammer.js/hammer.js');
+        app.import('vendor/hammer.js');
```

Comment:
the final build output of a users app in prod would minify this, fyi - but all good, not blocking